HW5/src/js/task1.js

```diff
@@ -0,0 +1,34 @@
+var date = new Date().getHours();
+var login = prompt('Enter your login');
+
+function checkEmptyDate(el) {
+    if (el === null) {
+        alert('Отменено');
+    }
+    else if (el === ""){
+        alert('Отменено')
+    }
+}
+checkEmptyDate(login);
+
+if (login.length < 4){
+    alert('I dont know any users having name length less than 4 symbols');
+}
+
+else if(login === "User"){
+    var password = prompt('Enter your password:');
+    checkEmptyDate(password);
+    if (password === 'SuperUser') {
+        if (date < 20){
```

Comment:
you can use trinary like dat < 20 ? alert('good day') : alert('Good Evening');

Reply:
+